README.md

```diff
@@ -10,7 +10,7 @@
 [![PyPI](https://img.shields.io/pypi/v/csp.svg?style=flat)](https://pypi.python.org/pypi/csp)
 [![License](https://img.shields.io/badge/license-Apache--2.0-green)](https://github.com/Point72/csp/LICENSE)
 [![Build Status](https://github.com/Point72/csp/actions/workflows/build.yml/badge.svg)](https://github.com/Point72/csp/actions/workflows/build.yml)
-[![Python Versions](https://img.shields.io/badge/python-3.8_%7C_3.9_%7C_3.10_%7C_3.11-blue)](https://github.com/Point72-OSPO/csp/blob/pyproject.toml)
+[![Python Versions](https://img.shields.io/badge/python-3.8_%7C_3.9_%7C_3.10_%7C_3.11-blue)](https://github.com/Point72/csp/blob/pyproject.toml)
```

Comment:
https://github.com/Point72/csp/blob/pyproject.toml isn't really a useful place to link to.

Why not drop the hyperlink entirely?

```suggestion
![Python Versions](https://img.shields.io/badge/python-3.8_%7C_3.9_%7C_3.10_%7C_3.11-blue)
```

Reply:
it is a hyperlink regardless, if you dont link it somewhere else it just links to the image

Reply:
I think @ngoldbaum meant that the link to pyproject.toml was also broken, i.e. it should be
[https://github.com/Point72/csp/blob/main/pyproject.toml](https://github.com/Point72/csp/blob/main/pyproject.toml)
not
[https://github.com/Point72/csp/blob/pyproject.toml](https://github.com/Point72/csp/blob/pyproject.toml)
Unless that's intentional...